bool req
---
geometry_msgs/Point lambert_ref
